e2e/tests/upgrades/upgrade_test.go

```diff
@@ -680,18 +679,6 @@ func (s *UpgradeTestSuite) RegisterInterchainAccount(ctx context.Context, chain
 	s.AssertTxSuccess(txResp)
 }
 
-// getICAVersion returns the version which should be used in the MsgRegisterAccount broadcast from the
-// controller chain.
-func getICAVersion(chainAVersion, chainBVersion string) string {
```

Comment:
this is now unused?

Reply:
Yup! Removed here https://github.com/cosmos/ibc-go/pull/3667#discussion_r1206468677